### ISSUE TITLE
Unable to add postprocessors using append

### ISSUE THREAD

**Issue by graipher** [ganga-devs/ganga#270]
**'GangaList' object has no attribute 'order'**

Ways to reproduce error:
```
$ SetupProject ganga --dev v601r16 && lhcb-proxy-init
$ ganga
Ganga In [1]: j = Job()
Ganga In [2]: j.postprocessors = [FileChecker()]
Ganga In [3]: j.postprocessors.append(FileChecker())
```
which dies:

```
Ganga.Runtime.bootstrap : ERROR Error: 'GangaList' object has no attribute 'order'
Ganga.Runtime.bootstrap : ERROR Unknown/Unexpected ERROR!!
---------------------------------------------------------------------------
AttributeError Traceback (most recent call last)
/afs/cern.ch/lhcb/software/DEV/GANGA/GANGA_v601r16/install/ganga/bin/ganga in <module>()
----> 1 j.postprocessors.append(FileChecker())

/afs/cern.ch/lhcb/software/DEV/GANGA/GANGA_v601r16/install/ganga/python/Ganga/GPIDev/Adapters/IPostProcessor.pyc in append(self, value)
     87 self.addProcess(value)
     88 self.process_objects = sorted(
---> 89 self.process_objects, key=lambda process: process.order)
     90
     91 def remove(self, value):

/afs/cern.ch/lhcb/software/DEV/GANGA/GANGA_v601r16/install/ganga/python/Ganga/GPIDev/Adapters/IPostProcessor.pyc in <lambda>(process)
     87 self.addProcess(value)
     88 self.process_objects = sorted(
---> 89 self.process_objects, key=lambda process: process.order)
     90
     91 def remove(self, value):

AttributeError: 'GangaList' object has no attribute 'order'
```

**Comment by drmarkwslater** [comment]
We'll try to get this in for 6.1.18 (6.1.17 is ~frozen at the moment). Thanks for reporting!

**Comment by egede** [comment]
@graipher Solution is part of #179. This should work before that one is merged.

**Comment by drmarkwslater** [comment]
This was fixed by #384
